### PR TITLE
chore(swamp): replace cosmos prefix with celestia in swamp tests

### PR DESCRIPTION
**Pull request by vgonkivs**
## Overview

Use "celestia" prefix for accounts instead of "cosmos" in swamp tests

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

**Comment by Wondertan** [comment]
Does it fixes tests on mac? Also the question why didn't it work before

File: nodebuilder/tests/swamp/swamp.go [hunk 1]
```diff
@@ -31,6 +34,13 @@ import (
 	"github.com/celestiaorg/celestia-node/nodebuilder/state"
 )
 
+func init() {
+	cfg := sdk.GetConfig()
+	cfg.SetBech32PrefixForAccount(app.Bech32PrefixAccAddr, app.Bech32PrefixAccPub)
+	cfg.SetBech32PrefixForValidator(app.Bech32PrefixValAddr, app.Bech32PrefixValPub)
+	cfg.Seal()
+}
```
Comment (Wondertan) [inline nodebuilder/tests/swamp/swamp.go:42]: Let's move this init to the core pkg. This way we always use the right prefix everywhere including swamp. Also if the node is going to be used as a library, then users won't need to do this as well.

**Review by Wondertan — CHANGES_REQUESTED** [review]

File: nodebuilder/tests/swamp/swamp.go (hunk 1 above)
Comment (vgonkivs) [reply nodebuilder/tests/swamp/swamp.go:42]: we are already using it here https://github.com/celestiaorg/celestia-node/blob/311d9dc757f716289978da23913a72fe01c99ede/cmd/celestia/main.go#L18-L21. So, we have to add these lines only for swamp tests

**Review by distractedm1nd — DISMISSED** [review]

**Review by renaynay — APPROVED** [review]

File: core/fetcher.go [hunk 2]
```diff
@@ -17,6 +20,15 @@ var (
...
 )
 
+func init() {
+	// This is necessary to ensure that the account addresses are correctly prefixed
+	// as in the celestia application.
+	cfg := sdk.GetConfig()
+	cfg.SetBech32PrefixForAccount(app.Bech32PrefixAccAddr, app.Bech32PrefixAccPub)
+	cfg.SetBech32PrefixForValidator(app.Bech32PrefixValAddr, app.Bech32PrefixValPub)
+	cfg.Seal()
+}
```
Comment (Wondertan) [inline core/fetcher.go:30]: Can we make a separate file for this? Fetcher is not a good place.

**Review by distractedm1nd — APPROVED** [review]

**Review by Wondertan — APPROVED** [review]

**Review by renaynay — APPROVED** [review]

**Pull request merged**